guvnor-organizationalunit-manager/src/main/java/org/guvnor/organizationalunit/manager/client/editor/popups/EditOrganizationalUnitPopup.ui.xml

```diff
@@ -23,7 +23,7 @@
 
     <b:Form type="HORIZONTAL">
         <b:FieldSet>
-            <b:Legend>Organizational Unit Information</b:Legend>
+            <b:Legend text='{i18n.OrganizationalUnitInformation}'></b:Legend>
             <b:FormGroup ui:field="nameGroup">
                 <b:FormLabel for="name" addStyleNames="col-md-4">
                     Name
```

Comment:
There are still some non-i18n'ed constants... ("Name", "Default Group ID", "Owner" in ```EditOrganizationalUnitPopup.ui.xml```)